plugins/chat/assets/javascripts/discourse/components/chat/message-creator/lib/chatables-loader.js

```diff
@@ -74,11 +74,13 @@ export default class ChatablesLoader {
         let chatable;
         if (channel.chatable?.users?.length === 1) {
           chatable = ChatChatable.createUser(channel.chatable.users[0]);
-          chatable.tracking = this.#injectTracking(chatable);
+          chatable.unread_thread_count =
```

Comment:
This failed to populate tracking data for 1-1 DMs because the chatable type is "user". However, we don't need to try and inject here anyway since we already have the channel data. I've also added the unread threads count here so that we can use it within the chat launcher to show unread threads, as we won't have the channel at that point for 1-1 chats.